### DISCUSSION TITLE
افزودن پروتکل هیستریا

### DISCUSSION THREAD

**Discussion by elrazez** [hiddify/Hiddify-Manager#328] [Ideas ایدهها]
سلام به همه
پروتکل هیستریا خیلی بهتر از v2ray داره جواب میده و اگه بتونیم کانفیگ‌هاشو داخل هیدیفای هم داشته باشیم فوق‌العاده می‌شه.

[https://github.com/apernet/hysteria](https://github.com/apernet/hysteria)

**Comment by zZedix** [comment]
طبق گزارشات داخل ایران این سرویس به سرعت فیلتر میشه ! علاوه بر اون مشکل این سرویس اینه که نرم افزار هایی که این نوع رو پشتیبانی میکنند داخل پلتفرم های مختلف خیلی کم هستند

**Reply by khak313** [reply]
تست کردین توی بلند مدت؟

**Reply by zZedix** [reply]
خیر ! چون client نداره اصلا ! من روی مک و ios هستم نداره اما خیلی جاها اعلام شده که سرعت بالاست اما خیلی سریع فیلتر میشه

**Reply by khak313** [reply]
کلاینت که هست

**Comment by zZedix** [comment]
طبق گزارشات داخل ایران این سرویس به سرعت فیلتر میشه ! علاوه بر اون مشکل این سرویس اینه که نرم افزار هایی که این نوع رو پشتیبانی میکنند داخل پلتفرم های مختلف خیلی کم هستند

**Comment by old-joker** [comment]
it seems good and speed is high 
have multiple client for all platforms
Matsuri , Shadowrocket , SagerNet
